back/engines/commercial/bulk_import_ideas/app/services/bulk_import_ideas/side_fx_bulk_import_service.rb

```diff
@@ -5,10 +5,19 @@ class SideFxBulkImportService
     def after_success(user, phase, model, format, items, users)
       options = { payload: { model: model, format: format, items_created: items.count, users_created: users.count } }
       activity_object = phase
+
+      project = Phase.find(phase).project
+      project_analyses = Analysis::Analysis.where(project: project) 
+      phase_analyses = Analysis::Analysis.where(phase: phase)  
+      analyses = (project_analyses + phase_analyses).uniq
+
       if items.count == 0
         LogActivityJob.perform_later activity_object, 'bulk_import_started', user, Time.now.to_i, options
       else
         LogActivityJob.perform_later activity_object, 'bulk_import_succeeded', user, items.last&.created_at, options
+        analyses.each do |analysis|
```

Comment:
I tried adding this as a patch but it didn't work. Not sure why, maybe because this is in a module also? I hope the current approach is acceptable.

Reply:
I think it's acceptable